data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -1048,7 +1050,7 @@ Sub-Network        |  L2  |  | TSN  |  | UDP  |
         See <xref target="control-management-requirements"/> for
         further discussion of these alternatives. In addition, <xref
         target="RFC2386"/> contains useful background information on
-        QoS-based routing, and <xref target="RFC5575"/> discusses a
+        QoS-based routing, and <xref target="RFC5575"/> <!-- <xref target="I-D.ietf-idr-rfc5575bis"/> --> discusses a
         specific mechanism used by BGP for traffic flow specification
         and policy-based routing.
       </t>

```

Comment:
In addition, <xref target="RFC2386"/> contains useful background information on QoS-based routing, and <xref target="RFC5575"/> (being updated by <xref target="I-D.ietf-idr-rfc5575bis"/>) discusses a specific mechanism used by BGP for traffic flow specification and policy-based routing.